homeassistant/components/sensibo/select.py

```diff
@@ -0,0 +1,131 @@
+"""Number platform for Sensibo integration."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from homeassistant.components.select import SelectEntity, SelectEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import SensiboDataUpdateCoordinator
+from .entity import SensiboBaseEntity
+
+
+@dataclass
+class SensiboSelectDescriptionMixin:
+    """Mixin values for Sensibo entities."""
+
+    remote_key: str
+    remote_options: str
+
+
+@dataclass
+class SensiboSelectEntityDescription(
+    SelectEntityDescription, SensiboSelectDescriptionMixin
+):
+    """Class describing Sensibo Number entities."""
+
+
+SELECT_TYPES = (
+    SensiboSelectEntityDescription(
+        key="horizontalSwing",
+        remote_key="horizontal_swing_mode",
+        remote_options="horizontal_swing_modes",
+        name="Horizontal Swing",
+        icon="mdi:air-conditioner",
+    ),
+    SensiboSelectEntityDescription(
+        key="light",
+        remote_key="light_mode",
+        remote_options="light_modes",
+        name="Light",
+        icon="mdi:flashlight",
+    ),
+)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up Sensibo number platform."""
+
+    coordinator: SensiboDataUpdateCoordinator = hass.data[DOMAIN][entry.entry_id]
+
+    async_add_entities(
+        SensiboSelect(coordinator, device_id, description)
+        for device_id, device_data in coordinator.data.parsed.items()
+        for description in SELECT_TYPES
+        if device_data["hvac_modes"]
+        and "horizontalSwing" in device_data["full_features"]
```

Comment:
It looks like this should be checking `light` for the light description. 
```suggestion
        and description.key in device_data["full_features"]
```